app-ios/Sources/ProfileCardFeature/Input/ProfileCardInputView.swift

```diff
@@ -0,0 +1,105 @@
+import SwiftUI
+import CommonComponents
+import ComposableArchitecture
+import Theme
+
+public struct ProfileCardInputView: View {
+    @Bindable private var store: StoreOf<ProfileCardInputReducer>
+
+    public init(store: StoreOf<ProfileCardInputReducer>) {
+        self.store = store
+    }
+
+    public var body: some View {
+        ScrollView {
+            VStack(spacing: 32) {
+                Text("Let's create a profile card to introduce yourself at events or on social media!", bundle: .module)
+                    .textStyle(.bodyLarge)
+                    .foregroundStyle(AssetColors.Surface.onSurfaceVariant.swiftUIColor)
+
+                ProfileCardInputTextField(
+                    title: String(localized: "Nickname", bundle: .module),
+                    text: .init(
+                        get: {
+                            store.nickname
+                        }, set: {
+                            store.send(.view(.nicknameChanged($0)))
+                        }
+                    )
+                )
```

Comment:
Right now you create the ProfileCardInputTextField as a component, but I think it would be more convenient to have the raw TextField available in this View if I want to manage the focus using FocusState in a future improvement.
```suggestion
        VStack(alignment: .leading) {
            Text(String(localized: "Nickname", bundle: .module))
                .textStyle(.titleMedium)
                .foregroundStyle(.white)

            TextField(placeholder, text: .init(...))
                .padding(.horizontal, 16)
                .padding(.vertical, 4)
                .frame(height: 56)
                .overlay(
                    RoundedRectangle(cornerRadius: 4)
                        .stroke(
                            AssetColors.Outline.outline.swiftUIColor,
                            style: StrokeStyle(lineWidth: 1)
                        )
                )
        }
```

However, I do not know if it is possible to control the focus without giving the TextField a focus modifier directly.
If it can be managed without directly giving focus modifier, then don't mind about this suggestion.

Reply:
There is no need to worry about focus at this point, so leave it as is. Thanks for the suggestion!